### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by cap-bots**
:robot: I have created a release *beep* *boop*
---

<details><summary>db-service: 1.9.0</summary>

### Added

* Add missing `func` cqn structures ([#629](https://github.com/cap-js/cds-dbs/issues/629)) ([9d7539a](https://github.com/cap-js/cds-dbs/commit/9d7539ab0fc7e70a6a00c0bd9cb4b3e362976e16))

### Fixed

* **`order by`:** reject non-fk traversals of own columns in order by ([#599](https://github.com/cap-js/cds-dbs/issues/599)) ([3288d63](https://github.com/cap-js/cds-dbs/commit/3288d63f0ee6a96580a3b2138ecb24a944371cf1))
* Align all quote functions with @sap/cds-compiler ([#619](https://github.com/cap-js/cds-dbs/issues/619)) ([42e9828](https://github.com/cap-js/cds-dbs/commit/42e9828baf11ec55281ea634ce56ce93e6741b91))
* assign artificial alias if selecting from anonymous subquery ([#608](https://github.com/cap-js/cds-dbs/issues/608)) ([e1a7711](https://github.com/cap-js/cds-dbs/commit/e1a77119f0a5241cfe4f50a37a473f2325ba5bde))
* avoid spread operator ([#630](https://github.com/cap-js/cds-dbs/issues/630)) ([a39fb65](https://github.com/cap-js/cds-dbs/commit/a39fb65f9419fe60e0324741039d004b40082903))
* flat update with arbitrary where clauses ([#598](https://github.com/cap-js/cds-dbs/issues/598)) ([f108798](https://github.com/cap-js/cds-dbs/commit/f108798c6c8035f9cdd0b9c6b8f334f1454c2faa))
* improved `=` and `!=` with val `null` for HANA and Postgres ([#626](https://github.com/cap-js/cds-dbs/issues/626)) ([cbcfe3b](https://github.com/cap-js/cds-dbs/commit/cbcfe3b15e8ebcf7e844dc5406e4bc228d4c94c9))
* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
* multiple result responses ([#602](https://github.com/cap-js/cds-dbs/issues/602)) ([bf0bed4](https://github.com/cap-js/cds-dbs/commit/bf0bed4549fe816e35481b0c9a7547a522a5a593))
* only consider persisted columns for simple operations ([#592](https://github.com/cap-js/cds-dbs/issues/592)) ([6e31bda](https://github.com/cap-js/cds-dbs/commit/6e31bda1bb15b1770b75c8971773806a26f7d452))

### Changed

* require `&gt;= sap/cds-compiler@4.9` ([f4d09e2](https://github.com/cap-js/cds-dbs/commit/f4d09e27c3b07dd88925e196aefc1087d8357f7a))
* require `&gt;= sap/cds@7.9.0` ([#627](https://github.com/cap-js/cds-dbs/issues/627)) ([f4d09e2](https://github.com/cap-js/cds-dbs/commit/f4d09e27c3b07dd88925e196aefc1087d8357f7a))
</details>

<details><summary>sqlite: 1.7.0</summary>

### Added

* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))

### Fixed

* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
* **hana:** Remove encoding from hana-client streams ([#623](https://github.com/cap-js/cds-dbs/issues/623)) ([fed8f6f](https://github.com/cap-js/cds-dbs/commit/fed8f6f36c5d97b664852a79050ce0a5e35a5c6d))
* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
</details>

<details><summary>postgres: 1.8.0</summary>

### Added

* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))

### Fixed

* Align all quote functions with @sap/cds-compiler ([#619](https://github.com/cap-js/cds-dbs/issues/619)) ([42e9828](https://github.com/cap-js/cds-dbs/commit/42e9828baf11ec55281ea634ce56ce93e6741b91))
* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
</details>

<details><summary>hana: 0.3.0</summary>

### Added

* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))

### Fixed

* Add multi `concat` function to `@cap-js/hana` ([#624](https://github.com/cap-js/cds-dbs/issues/624)) ([df436fe](https://github.com/cap-js/cds-dbs/commit/df436fec3e137dee81f4a5ed69e551fc7c92700e))
* Align all quote functions with @sap/cds-compiler ([#619](https://github.com/cap-js/cds-dbs/issues/619)) ([42e9828](https://github.com/cap-js/cds-dbs/commit/42e9828baf11ec55281ea634ce56ce93e6741b91))
* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
* Disconnect HANA tenant when deleted ([#589](https://github.com/cap-js/cds-dbs/issues/589)) ([a107db9](https://github.com/cap-js/cds-dbs/commit/a107db9dc0ce610ba07a4562e94cfd22a9f8c182))
* **hana:** Align not found behavior in @cap-js/hana ([#603](https://github.com/cap-js/cds-dbs/issues/603)) ([54d2efb](https://github.com/cap-js/cds-dbs/commit/54d2efb00cfa4b5f188dc01bd350f3ccaca8986b))
* **hana:** Allow custom fuzzy search cqn ([#620](https://github.com/cap-js/cds-dbs/issues/620)) ([80383f0](https://github.com/cap-js/cds-dbs/commit/80383f0e5aa3a81592e804c02ce6253bd4e7d16e))
* **hana:** Allow HANA to use != and == inside xpr combinations ([#607](https://github.com/cap-js/cds-dbs/issues/607)) ([c578e9f](https://github.com/cap-js/cds-dbs/commit/c578e9fd530ddd0de6e693b2bfe777935e935772))
* **hana:** Reference column alias in order by ([#615](https://github.com/cap-js/cds-dbs/issues/615)) ([7cd3a26](https://github.com/cap-js/cds-dbs/commit/7cd3a26943e9babdee385916d33e6ae16f48bd5d))
* **hana:** Remove encoding from hana-client streams ([#623](https://github.com/cap-js/cds-dbs/issues/623)) ([fed8f6f](https://github.com/cap-js/cds-dbs/commit/fed8f6f36c5d97b664852a79050ce0a5e35a5c6d))
* **hana:** Support associations with static values ([#604](https://github.com/cap-js/cds-dbs/issues/604)) ([05babcf](https://github.com/cap-js/cds-dbs/commit/05babcf7581b651b74b3f5eb1ebcb45dea706b06))
* improved `=` and `!=` with val `null` for HANA and Postgres ([#626](https://github.com/cap-js/cds-dbs/issues/626)) ([cbcfe3b](https://github.com/cap-js/cds-dbs/commit/cbcfe3b15e8ebcf7e844dc5406e4bc228d4c94c9))
* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: hana/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2.1](https://github.com/cap-js/cds-dbs/compare/hana-v0.2.0...hana-v0.2.1) (2024-04-19)
+
+
+### Fixed
+
+* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
```
Comment (patricebender) [inline hana/CHANGELOG.md:12]: If possible we should try to  avoid making changes in other DBs while fixing bugs for another. It is confusing that there is a "Postgres" note in the changelog for SAP HANA.

File: hana/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@
...
+
+
+### Added
+
+* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))
+
+
+### Fixed
+
+* Add multi `concat` function to `@cap-js/hana` ([#624](https://github.com/cap-js/cds-dbs/issues/624)) ([df436fe](https://github.com/cap-js/cds-dbs/commit/df436fec3e137dee81f4a5ed69e551fc7c92700e))
```
Comment (patricebender) [inline hana/CHANGELOG.md:17]:
```suggestion
* Add multi `concat` function ([#624](https://github.com/cap-js/cds-dbs/issues/624)) ([df436fe](https://github.com/cap-js/cds-dbs/commit/df436fec3e137dee81f4a5ed69e551fc7c92700e))
```

File: hana/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,30 @@
...
+* Align all quote functions with @sap/cds-compiler ([#619](https://github.com/cap-js/cds-dbs/issues/619)) ([42e9828](https://github.com/cap-js/cds-dbs/commit/42e9828baf11ec55281ea634ce56ce93e6741b91))
+* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
+* Disconnect HANA tenant when deleted ([#589](https://github.com/cap-js/cds-dbs/issues/589)) ([a107db9](https://github.com/cap-js/cds-dbs/commit/a107db9dc0ce610ba07a4562e94cfd22a9f8c182))
+* **hana:** Align not found behavior in @cap-js/hana ([#603](https://github.com/cap-js/cds-dbs/issues/603)) ([54d2efb](https://github.com/cap-js/cds-dbs/commit/54d2efb00cfa4b5f188dc01bd350f3ccaca8986b))
```
Comment (patricebender) [inline hana/CHANGELOG.md:21]:
```suggestion
* Align "not found" behavior ([#603](https://github.com/cap-js/cds-dbs/issues/603)) ([54d2efb](https://github.com/cap-js/cds-dbs/commit/54d2efb00cfa4b5f188dc01bd350f3ccaca8986b))
```

File: hana/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,30 @@
...
+* **hana:** Allow custom fuzzy search cqn ([#620](https://github.com/cap-js/cds-dbs/issues/620)) ([80383f0](https://github.com/cap-js/cds-dbs/commit/80383f0e5aa3a81592e804c02ce6253bd4e7d16e))
+* **hana:** Allow HANA to use != and == inside xpr combinations ([#607](https://github.com/cap-js/cds-dbs/issues/607)) ([c578e9f](https://github.com/cap-js/cds-dbs/commit/c578e9fd530ddd0de6e693b2bfe777935e935772))
+* **hana:** Reference column alias in order by ([#615](https://github.com/cap-js/cds-dbs/issues/615)) ([7cd3a26](https://github.com/cap-js/cds-dbs/commit/7cd3a26943e9babdee385916d33e6ae16f48bd5d))
+* **hana:** Remove encoding from hana-client streams ([#623](https://github.com/cap-js/cds-dbs/issues/623)) ([fed8f6f](https://github.com/cap-js/cds-dbs/commit/fed8f6f36c5d97b664852a79050ce0a5e35a5c6d))
+* **hana:** Support associations with static values ([#604](https://github.com/cap-js/cds-dbs/issues/604)) ([05babcf](https://github.com/cap-js/cds-dbs/commit/05babcf7581b651b74b3f5eb1ebcb45dea706b06))
```
Comment (patricebender) [inline hana/CHANGELOG.md:26]:
```suggestion
* Allow custom fuzzy search cqn ([#620](https://github.com/cap-js/cds-dbs/issues/620)) ([80383f0](https://github.com/cap-js/cds-dbs/commit/80383f0e5aa3a81592e804c02ce6253bd4e7d16e))
* Allow HANA to use != and == inside xpr combinations ([#607](https://github.com/cap-js/cds-dbs/issues/607)) ([c578e9f](https://github.com/cap-js/cds-dbs/commit/c578e9fd530ddd0de6e693b2bfe777935e935772))
* Reference column alias in order by ([#615](https://github.com/cap-js/cds-dbs/issues/615)) ([7cd3a26](https://github.com/cap-js/cds-dbs/commit/7cd3a26943e9babdee385916d33e6ae16f48bd5d))
* Remove encoding from hana-client streams ([#623](https://github.com/cap-js/cds-dbs/issues/623)) ([fed8f6f](https://github.com/cap-js/cds-dbs/commit/fed8f6f36c5d97b664852a79050ce0a5e35a5c6d))
* Support associations with static values ([#604](https://github.com/cap-js/cds-dbs/issues/604)) ([05babcf](https://github.com/cap-js/cds-dbs/commit/05babcf7581b651b74b3f5eb1ebcb45dea706b06))
```

File: hana/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,30 @@
...
+* improved `=` and `!=` with val `null` for HANA and Postgres ([#626](https://github.com/cap-js/cds-dbs/issues/626)) ([cbcfe3b](https://github.com/cap-js/cds-dbs/commit/cbcfe3b15e8ebcf7e844dc5406e4bc228d4c94c9))
```
Comment (patricebender) [inline hana/CHANGELOG.md:27]:
```suggestion
* improved `=` and `!=` with val `null` ([#626](https://github.com/cap-js/cds-dbs/issues/626)) ([cbcfe3b](https://github.com/cap-js/cds-dbs/commit/cbcfe3b15e8ebcf7e844dc5406e4bc228d4c94c9))
```

File: postgres/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,21 @@
...
+
+* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))
+
+
+### Fixed
+
+* Align all quote functions with @sap/cds-compiler ([#619](https://github.com/cap-js/cds-dbs/issues/619)) ([42e9828](https://github.com/cap-js/cds-dbs/commit/42e9828baf11ec55281ea634ce56ce93e6741b91))
+* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
+* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
+* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
```
Comment (patricebender) [inline postgres/CHANGELOG.md:20]:
```suggestion
* Use json datatype for `INSERT` ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
```

File: sqlite/CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,20 @@
...
+
+### Added
+
+* select decimals as strings if cds.env.features.string_decimals ([#616](https://github.com/cap-js/cds-dbs/issues/616)) ([39addbf](https://github.com/cap-js/cds-dbs/commit/39addbfe01da757d86a4d65e62eda86e59fc9b87))
+
+
+### Fixed
+
+* Change `sql` property to `query` for errors ([#611](https://github.com/cap-js/cds-dbs/issues/611)) ([585577a](https://github.com/cap-js/cds-dbs/commit/585577a9817e7749fb71958c26c4bfa20981c663))
+* **hana:** Remove encoding from hana-client streams ([#623](https://github.com/cap-js/cds-dbs/issues/623)) ([fed8f6f](https://github.com/cap-js/cds-dbs/commit/fed8f6f36c5d97b664852a79050ce0a5e35a5c6d))
```
Comment (patricebender) [inline sqlite/CHANGELOG.md:18]:
```suggestion
```

File: hana/CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,30 @@
...
+* Improved placeholders and limit clause ([#567](https://github.com/cap-js/cds-dbs/issues/567)) ([d5d5dbb](https://github.com/cap-js/cds-dbs/commit/d5d5dbb7219bcef6134440715cf756fdd439f076))
+* Use json datatype for Postgres insert ([#582](https://github.com/cap-js/cds-dbs/issues/582)) ([f1c9c89](https://github.com/cap-js/cds-dbs/commit/f1c9c89036a7f8e4709c67d713d06926630aa36d))
```
Comment (patricebender) [inline hana/CHANGELOG.md:29]:
```suggestion
```

**Review by patricebender — APPROVED** [review]

**Review by johannes-vogel — APPROVED** [review]

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/db-service-v1.9.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/sqlite-v1.7.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/postgres-v1.8.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/hana-v0.3.0 :sunflower:

**Pull request merged**